coderd/database/migrations/000391_aibridge_interceptions_key_id.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE aibridge_interceptions ADD COLUMN api_key_id text;
```

Comment:
This needs to be a not-nullable `UUID` column, with a foreign key to the `api_keys` table please.

Reply:
`id` column in `api_keys` table is text. I'd rather keep the same data type.

I didn't add foreign key constraint because I can image scenario where some key is removed but audit logs / interceptions should stay or race condition when key is removed while it is being used so key verification passes but interception recording fails.